desc/compute/_qs.py

```diff
@@ -301,6 +422,48 @@ def _B_modes(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="Boozer transform prefactor modes norm",
+    label="\\nu_{mn} = (\\zeta_{B} - \\zeta)_{mn}",
+    units="rad",
+    units_long="radians",
+    description="Inner product norm for boozer modes basis.",
+    dim=1,
+    params=[],
+    transforms={"B": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+)
+def _boozer_prefactor_modes_norm(params, transforms, profiles, data, **kwargs):
+    # norm is 1 if m=n=0, 2 if m=0 or n=0, 4 if m!=0 and n!=0
+    norm = 2 ** (3 - jnp.sum((transforms["B"].basis.modes == 0), axis=1))
+    data["Boozer transform prefactor modes norm"] = norm
+    return data
+
+
+@register_compute_fun(
+    name="Boozer transform prefactor",
```

Comment:
minor point but I wouldn't call the whole matrix a prefactor, I would think that the matrix really is the transform (in the same way that the dft can be written as a matrix operation). Maybe `Boozer transform matrix`?

Reply:
thanks, this was just a placeholder name anyways